src/app/communities/communities-edit.component.html

```diff
@@ -0,0 +1,50 @@
+<form [formGroup]="communityForm" class="communities-edit-form">
+  <div *ngIf="errorMessage" class="communities-edit-error">{{errorMessage}}</div>
+  <div fxLayout="column">
+    <div fxFlex>
+      <mat-form-field class="communities-edit-form-field">
+        <input matInput id="communities-edit-title" placeholder="Title" formControlName="title" [required]="true">
+      </mat-form-field>
+    </div>
+    <div fxFlex>
+      <mat-form-field class="communities-edit-form-field">
+        <textarea matInput placeholder="Description" formControlName="description" rows="8"></textarea>
+      </mat-form-field>
+    </div>
+  </div>
+  <mat-divider></mat-divider>
+  <div>
+    <button mat-raised-button class="communities-link-toolbar" (click)="addLink()">Add link</button>
+  </div>
+  <div formArrayName="links">
+    <mat-card *ngFor="let link of linkList.controls; let i = index;" class="communities-link-card">
+      <mat-card-content [formGroupName]="i">
+        <div fxLayout="row">
+          <div fxFlex="25%">
+            <mat-form-field class="communities-edit-form-field">
+              <input matInput  placeholder="Name" formControlName="name" [required]="true">
+            </mat-form-field>
+          </div>
+          <div fxFlex="70%">
+            <mat-form-field class="communities-edit-form-field communities-edit-href">
+              <input matInput  placeholder="Href" formControlName="href" [required]="true">
```

Comment:
I think it is better to have here a placeholder as `hyperreference` or `link` instead of `href`. A common user can be confused with such a word.

Reply:
done